frontend/src/lib/taxonomy.tsx

```diff
@@ -926,6 +926,38 @@ export function isPostHogProp(key: string): boolean {
 export type PropertyKey = string | null | undefined
 export type PropertyType = 'event' | 'element'
 
+// copy from https://github.com/PostHog/posthog/blob/29ac8d6b2ba5de4b65a148136b681b8e52e20429/plugin-server/src/utils/db/utils.ts#L44
+const eventToPersonProperties = new Set([
```

Comment:
is there a module where I can reuse this between plugin-server and frontend? I don't like to dupe stuff.

Reply:
Unfortunately not…

Reply:
Not yet at least – I'm sure there are ways, a basic one being symlinks. But we don't have any code shared between these two at this moment

Reply:
will keep the dupe for now then, the comments help to keep it in sync, otherwise I'd need a new file just for that.